app/controllers/application_templates_controller.rb

```diff
@@ -19,6 +19,11 @@ def new
   def show
   end
 
+  def toggle_active
+    @template.toggle!(:active)
```

Comment:
Because bang saves the updated object, "toggle" by itself does not. I _think_ I would have to return self if I did not use bang.